AssertionError [ERR_ASSERTION]: logs do not look right: 1

https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/kubeflow_pipelines/430/presubmit-e2e-test/630/

```
run-frontend-integration-tests:	․․․․․․Handling connection for 3000
run-frontend-integration-tests:	F22:40:04.744 INFO [ActiveSessions$1.onStop] - Removing session 0fba00219d7e59cb2f76e1d38031312a (org.openqa.selenium.chrome.ChromeDriverService)
run-frontend-integration-tests:	15 passing (34.10s)
run-frontend-integration-tests:	1 failing
run-frontend-integration-tests:	1) deploy helloworld sample run shows logs from node:
run-frontend-integration-tests:	logs do not look right: 1
run-frontend-integration-tests:	running chrome
run-frontend-integration-tests:	AssertionError [ERR_ASSERTION]: logs do not look right: 1
run-frontend-integration-tests:	    at Context.it (/src/helloworld.spec.js:169:5)
run-frontend-integration-tests:	    at new Promise (<anonymous>)
run-frontend-integration-tests:	    at new F (/src/node_modules/core-js/library/modules/_export.js:36:28)
run-frontend-integration-tests:	Wrote xunit report "junit_FrontendIntegrationTestOutput.xml" to [./].
run-frontend-integration-tests:	npm ERR! Test failed.  See above for more details.
```

--- a/test/frontend-integration-test/helloworld.spec.js
+++ b/test/frontend-integration-test/helloworld.spec.js
@@ -165,9 +165,10 @@ describe('deploy helloworld sample run', () => {
   it('shows logs from node', () => {
     $('button=Logs').click();
     $('#logViewer').waitForVisible();
-    const logs = $('#logViewer').getText();
-    assert(logs.indexOf(outputParameterValue + ' from node: ') > -1,
-      'logs do not look right: ' + logs);
+    browser.waitUntil(() => {
+      const logs = $('#logViewer').getText();
+      return logs.indexOf(outputParameterValue + ' from node: ') > -1;
+    }, waitTimeout);
   });
   //TODO: enable this after we change the pipeline to a unique name such that deleting this
   // pipeline will not jeopardize the concurrent basic e2e tests.

